File: ios/ExpoPasteInputModule.swift
import ExpoModulesCore

public class ExpoPasteInputModule: Module {
  public func definition() -> ModuleDefinition {
    Name("ExpoPasteInput")
    
    View(ExpoPasteInputView.self) {
      Events("onPaste")

      Prop("disabled") { (view: ExpoPasteInputView, disabled: Bool) in
        view.disabled = disabled
      }
    }
  }
}

File: android/src/main/java/expo/modules/pasteinput/ExpoPasteInputModule.kt
package expo.modules.pasteinput

import expo.modules.kotlin.modules.Module
import expo.modules.kotlin.modules.ModuleDefinition

class ExpoPasteInputModule : Module() {
  override fun definition() = ModuleDefinition {
    Name("ExpoPasteInput")
    
    View(ExpoPasteInputView::class) {
      Events("onPaste")
    }
  }
}
